code/game/machinery/GBPvendor.dm

```diff
@@ -0,0 +1,128 @@
+
+/obj/machinery/GBP_vendor
+	name = "engineering point redemption"
+	desc = "Who's a good boy?"
+	icon = 'icons/obj/vending.dmi'
+	icon_state = "liberationstation"
+	density = 1
+	anchored = 1
+	var/GBP = 0
+	var/list/prize_list = list( //if you add something to this, please, for the love of god, use tabs and not spaces.
+		new /datum/data/GBP_equipment("Whiskey",				/obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey,		10),
+		new /datum/data/GBP_equipment("Cigar",				/obj/item/clothing/mask/cigarette/cigar/havana,							20),
+		new /datum/data/GBP_equipment("Soap",				/obj/item/weapon/soap/nanotrasen,										100),
+		new /datum/data/GBP_equipment("Insulated Gloves",				/obj/item/clothing/gloves/color/yellow,						100),
+		new /datum/data/GBP_equipment("Fulton Beacon",		/obj/item/fulton_core,													30),
+		new /datum/data/GBP_equipment("Fulton Pack",			/obj/item/weapon/extraction_pack,									200),
+		new /datum/data/GBP_equipment("Space Cash",			/obj/item/stack/spacecash/c1000,										200),
+		new /datum/data/GBP_equipment("50 metal sheets",			/obj/item/stack/sheet/metal/fifty,								250),
+		new /datum/data/GBP_equipment("50 glass sheets",			/obj/item/stack/sheet/glass/fifty,								250),
+		new /datum/data/GBP_equipment("50 glass sheets",			/obj/item/stack/sheet/cardboard/fifty,							250),
+		new /datum/data/GBP_equipment("Hardsuit Set",			/obj/item/clothing/suit/space/hardsuit,								500),
+		new /datum/data/GBP_equipment("Jetpack Upgrade Set",		/obj/item/weapon/tank/jetpack/suit,								750),
+		new /datum/data/GBP_equipment("Advanced Magboot Set",			/obj/item/clothing/shoes/magboots/advance,					1500),
+		new /datum/data/GBP_equipment("ERT Hardsuit Set",		/obj/item/clothing/suit/space/hardsuit/ert,							4000),
+		new /datum/data/GBP_equipment("Portal Gun Set",			/obj/item/weapon/gun/energy/wormhole_projector,						4000),
+		new /datum/data/GBP_equipment("Reactive Decoy Armor Set",		/obj/item/clothing/suit/armor/reactive/stealth,				6000),
+		new /datum/data/GBP_equipment("Cloaking Belt Set",		/obj/item/device/shadowcloak,										8000),
+		new /datum/data/GBP_equipment("Chrono Suit Set",		/obj/item/clothing/suit/space/chronos,								10000),
+		new /datum/data/GBP_equipment("WHAT HAVE YOU DONE... Set",		/obj/vehicle/space/speedbike/speedwagon,					30000),

```

Comment:
You know the N64 prize at Chuckie Cheeses? How absurd it seemed that you could a NINTENDO SIXTY-FOUR just for winning enough games of skeeball? Then you did the math in your head and realized how much time and money you'd have to spend playing skeeball to earn the 60,000 tickets it cost? 

I tested every one of these items and they are all reasonable for their cost. Plus, these aren't final rewards, these aren't even semi-final rewards, these are literally just placeholders to motivate engineers and therefore for live tests to reveal exactly how much power "motivated engineers" can pull during a typical round.